grails-app/services/au/org/ala/merit/ProjectService.groovy

```diff
@@ -600,6 +600,18 @@ class ProjectService  {
         result
     }
 
+    def cancelReport(String projectId, Map reportDetails) {
+        Map reportInformation = prepareReport(projectId, reportDetails)
+        if (reportInformation.error) {
+            return [success:false, error:reportInformation.error]
+        }
+
+        EmailTemplate emailTemplate = ((ProgramConfig)reportInformation.config).getReportReturnedTemplate()
```

Comment:
We need to create a new email template for cancelled reports, otherwise a confusing email will be sent to the user.

Reply:
I just removed the email template for now as per Blaz they don't need the notification otherwise will create a new one if they opted to have a notification.